pkg/upload/dsym.go

```diff
@@ -97,9 +103,13 @@ func ProcessDsym(
 
 					// Check if dsymPath exists before proceeding
 					if utils.Path(dsymPath).Validate() != nil {
-						// TODO: This will be toggled between Error and Warn with --ignore-missing-dwarf in near future
-						log.Error("Could not find dSYM with scheme '"+scheme+"' in expected location: "+utils.DisplayBlankIfEmpty(dsymPath)+"\n\n"+
-							"Check that the scheme correlates to the above dSYM location, try re-building your project or specify the dSYM path using --dsym-path\n", 1)
+						if ignoreMissingDwarf {
+							log.Warn("Could not find dSYM with scheme '" + scheme + "' in expected location: " + utils.DisplayBlankIfEmpty(dsymPath) + "\n\n" +
+								"Check that the scheme correlates to the above dSYM location, try re-building your project or specify the dSYM path using --dsym-path\n")
+						} else {
+							log.Error("Could not find dSYM with scheme '"+scheme+"' in expected location: "+utils.DisplayBlankIfEmpty(dsymPath)+"\n\n"+
+								"Check that the scheme correlates to the above dSYM location, try re-building your project or specify the dSYM path using --dsym-path\n", 1)
+						}
```

Comment:
On reflection, I don't think this is the scenario we're talking about for missing dwarf information.

I think we'd be OK just to validate the path and set it if it exists, otherwise don't set `dsymPath` and let it error later on.